node/tests/k8st/tests/test_bgp_advert.py

```diff
@@ -1,4 +1,3 @@
-# Copyright (c) 2018 Tigera, Inc. All rights reserved.
```

Comment:
I think this should stay, but be updated to say "2018-2024"

Reply:
Deleted this line by mistake, I'll add it back